streams/src/test/java/org/apache/kafka/streams/state/internals/MergedSortedCacheWrappedWindowStoreIteratorTest.java

```diff
@@ -85,4 +85,18 @@ public void shouldPeekNextKey() throws Exception {
         assertThat(iterator.peekNextKey(), equalTo(10L));
     }
 
-}
\ No newline at end of file
+    @Test
+    public void shouldPeekNextKey2() throws Exception {
```

Comment:
they both check different code paths, shouldPeekNextKey checks the code path here https://github.com/apache/kafka/pull/3016/files#diff-6ea5070fb98ce2270ef8a94a2eb14511L133 vs. shouldPeekNextKey2 checks the code path here https://github.com/apache/kafka/pull/3016/files#diff-6ea5070fb98ce2270ef8a94a2eb14511R137